pkg/devfile/parser/context/context.go

```diff
@@ -38,6 +38,29 @@ func NewDevfileCtx(path string) DevfileCtx {
 	}
 }
 
+// PopulateDevfile fills the DevfileCtx struct with relevant context info
+func (d *DevfileCtx) PopulateDevfile() (err error) {
```

Comment:
Does this function need to be exported? The documentation should also explain how it differs from `Populate`.

Reply:
I have changed this now